<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-06-21T18:23:06 IST" name="Suite" finished-at="2024-06-21T18:23:07 IST" duration-ms="69">
    <groups>
    </groups>
    <test started-at="2024-06-21T18:23:06 IST" name="Test" finished-at="2024-06-21T18:23:07 IST" duration-ms="69">
      <class name="testRunner.SlideTeam_Login_TC_1">
        <test-method signature="ST_TC_1(java.lang.String,java.lang.String)[pri:0, instance:testRunner.SlideTeam_Login_TC_1@cb0755b]" started-at="2024-06-21T18:23:06 IST" name="ST_TC_1" data-provider="data" finished-at="2024-06-21T18:23:07 IST" duration-ms="21" status="FAIL">
          <exception class="org.testng.internal.reflect.MethodMatcherException">
            <message>
              <![CDATA[
[public void testRunner.SlideTeam_Login_TC_1.ST_TC_1(java.lang.String,java.lang.String)] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: ST_TC_1([java.lang.String arg0, java.lang.String arg1])
Arguments: [(java.lang.String) Valid Deatils,(java.lang.String) testuser27thapril@gmail.com,(java.lang.String) 123456]]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.internal.reflect.MethodMatcherException: 
[public void testRunner.SlideTeam_Login_TC_1.ST_TC_1(java.lang.String,java.lang.String)] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: ST_TC_1([java.lang.String arg0, java.lang.String arg1])
Arguments: [(java.lang.String) Valid Deatils,(java.lang.String) testuser27thapril@gmail.com,(java.lang.String) 123456]
at org.testng.internal.reflect.DataProviderMethodMatcher.getConformingArguments(DataProviderMethodMatcher.java:46)
at org.testng.internal.Parameters.injectParameters(Parameters.java:915)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.internal.reflect.MethodMatcherException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ST_TC_1 -->
      </class> <!-- testRunner.SlideTeam_Login_TC_1 -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
